packages/cx-templates/doc/README.md

```diff
@@ -15,9 +15,11 @@ Using templates provides the following benefits:
 
 ## Brief Overview of Templates in the Test Site
 
-* [Generic Template](./Generic) - A basic page with 3 slots for content + breadcrumb.
-* [Product Detail Page](./ProductDetailPage) - A page oriented to showcase product and details about the specific product.
-* [Product Listing Page](./ProductListingPage) - A landinage page allowing the user to interact and filter to view avaible products.
+* [Generic Template](./Generic): A basic page with three slots for content and one for breadcrumbs.
+* [Product Detail Page](./ProductDetailPage): A page oriented to showcase a product and details
```

Comment:
I think we shoudl indicate somehow that these are not yet implemented

Reply:
Added a warning box. 👍